libs/ledger-live-common/src/families/cosmos/helpers.ts

```diff
@@ -13,3 +13,26 @@ export const getMainMessage = (messages: CosmosMessage[]): CosmosMessage => {
     .sort((a, b) => messagePriorities.indexOf(a.type) - messagePriorities.indexOf(b.type));
   return sortedTypes[0];
 };
+
+const order = unordered =>
+  Object.keys(unordered)
+    .sort()
+    .reduce((obj, key) => {
+      obj[key] = unordered[key];
+      return obj;
+    }, {});
+
+export const sortObjectKeysDeeply = object => {
+  for (let [key, value] of Object.entries(object)) {
+    if (Array.isArray(value)) {
+      const newArray = new Array();
+      for (const element of value) {
+        newArray.push(sortObjectKeysDeeply(element));
+      }
+      object[key] = newArray;
+    } else if (typeof value === "object") {
+      object[key] = sortObjectKeysDeeply(value);
+    }
+  }
+  return order(object);
+};
```

Comment:
No clever lodash that does this?
Maybe for `order` at least?

Reply:
Found
```
var ordered = {};   
_(Categories).keys().sort().each(function (key) {
  ordered[key] = Categories[key];
});

Categories = ordered;
```

```
_(obj).toPairs().sortBy(0).fromPairs().value()
```

Lib [lodash-keyarrange](https://www.npmjs.com/package/lodash-keyarrange)

Only the lib cover every sublevels

Do you have any preference ? I find my current solution to be ok-tier

Feel free to search more than me

Reply:
I had a quick look and didn't find anything great either, so your current version is ok to me :+1: 
Just see other my comment about safety on recursion.